irods/test/meta_test.py

```diff
@@ -121,6 +122,18 @@ def tearDown(self):
 
     from irods.test.helpers import create_simple_resc_hierarchy
 
+    def test_atomic_metadata_json_error__issue_606(self):
+        path = self.coll_path + "/atomic_meta_issue_606"
+        obj = self.sess.data_objects.create(path)
+        obj.unlink(force = True)
+        try:
+            obj.metadata.apply_atomic_operations(AVUOperation(operation="add", avu=iRODSMeta('a','b','c')))
+        except ex.iRODSException as e:
```

Comment:
Should we make some sort of assertion about the fact that an `Exception` ought to be raised here? Like, we're using a `try`-`except` here - which is fine - but maybe we should add an `else` to it that causes the test to fail. As it stands, the test will pass if *somehow* `obj` begins to exist again.

I know, unlikely-to-impossible... but... would be super airtight.

Reply:
Can do.

Reply:
This is done.